### PR TITLE
[DESIGN2-202-Chips] Chips Component - DSCO - Update color variables

### PR DESCRIPTION
**Pull request by kelbyhawn**
Swaps out color variables from `shared/color.scss` with Primitive colors from `primitiveColors.css` on the Chips component. 

👀 **DOTD note:** there might be Eyes diffs with this.

## Links
Jira ticket: [DESIGN2-202](https://codedotorg.atlassian.net/browse/DESIGN2-202)

## Testing story
Tested locally in Storybook

----

## Before - using `shared/colors.scss`
Before

## After - using `primitiveColors.css`
After

### CODE REVIEW

File: apps/src/componentLibrary/chips/chip.module.scss [hunk 1]
```diff
@@ -14,9 +14,9 @@
 
     .chip {
       display: inline-block;
-      background-color: $light_white;
-      border: 1px solid $light_black;
-      color: $light_black;
+      background-color: var(--neutral-base-white);
+      border: 1px solid var(--neutral-base-black);
+      color: var(--neutral-base-black);
```
Comment (kelbyhawn) [inline apps/src/componentLibrary/chips/chip.module.scss:19]: black

File: apps/src/componentLibrary/chips/chip.module.scss [hunk 2]
```diff
@@ -37,58 +37,58 @@
...
-        color: $light_white;
+        border: 1px solid var(--brand-teal-50);
+        background-color: var(--brand-teal-50);
+        color: var(--neutral-base-white);
       }
 
       // Hover styles
       &:hover {
-        background-color: $light_gray_100;
+        background-color: var(--neutral-gray-10);
```
Comment (kelbyhawn) [inline apps/src/componentLibrary/chips/chip.module.scss:47]: gray

File: apps/src/componentLibrary/chips/chip.module.scss [hunk 3]
```diff
@@ -37,58 +37,58 @@
...
 
         &:has(input[type="checkbox"]:checked) {
-          background-color: $light_primary_700;
+          background-color: var(--brand-teal-70);
```
Comment (kelbyhawn) [inline apps/src/componentLibrary/chips/chip.module.scss:50]: Screenshot 2024-12-11 at 10 37 07 AM

File: apps/src/componentLibrary/chips/chip.module.scss [hunk 4]
```diff
@@ -37,58 +37,58 @@
...
+        outline: 2px solid var(--brand-teal-50);
         outline-offset: 2px;
       }
 
       //Disabled styles
       &:has(input[type="checkbox"]:disabled) {
-        color: $light_gray_200;
-        border-color: $light_gray_200;
+        color: var(--neutral-gray-20);
+        border-color: var(--neutral-gray-20);
```
Comment (kelbyhawn) [inline apps/src/componentLibrary/chips/chip.module.scss:76]: Screenshot 2024-12-11 at 10 38 56 AM

**Review by levadadenys — APPROVED** [review]
LGTM! 🚀

**Pull request merged**